tests/components/downloader/test_config_flow.py

```diff
@@ -33,7 +33,7 @@ async def test_user_form(hass: HomeAssistant) -> None:
         assert result["type"] is FlowResultType.FORM
 
         with patch(
-            "homeassistant.components.downloader.config_flow.DownloaderConfigFlow._validate_input",
+            "os.path.isabs",
             side_effect=DirectoryDoesNotExist,
         ):
```

Comment:
Why don't we patch `os.path.isdir` to return False? That will raise the exception as well and makes the test less coupled